tests/reports/tests-reports-functions.php

```diff
@@ -422,6 +422,29 @@ public function test_parse_dates_for_range_with_last_month_range_should_return_t
 		$this->assertEqualSetsWithIndex( $expected, $result );
 	}
 
+	/**
+	 * @covers \EDD\Reports\parse_dates_for_range()
+	 * @group edd_dates
+	 */
+	public function test_parse_dates_for_range_with_overflow_last_month_range_should_return_those_dates() {
+		$overflow_day  = '2020-03-30';
+		$overflow_date = EDD()->utils->date( $overflow_day );
+
+		$expected = array(
+			'start' => $overflow_date->copy()->subMonthNoOverflow( 1 )->startOfMonth()->toDateTimeString(),
+			'end'   => $overflow_date->copy()->subMonthNoOverflow( 1 )->endOfMonth()->toDateTimeString(),
```

Comment:
Since we're passing in an explicit `$overflow_day`, would it make more sense and be possible for these `start` and `end` values to be the hard-coded dates we expect instead of using Carbon again?